std_msgs/Header header
geometry_msgs/Point pos  #  [mm]
geometry_msgs/Quaternion orientation  # quaternion
geometry_msgs/Vector3 rpy  # rpy [degree]
geometry_msgs/Vector3 linear_velocity  # [mm/s]
float64 fig  #fig
